statev2/src/replication/log_store.rs

```diff
@@ -37,6 +37,9 @@ pub const CONF_STATE_KEY: &str = "conf-state";
 /// The name of the snapshot metadata key in the KV store
 pub const SNAPSHOT_METADATA_KEY: &str = "snapshot-metadata";
 
+/// A marker for a placeholder in an unused variable
+pub const UNUSED: u64 = 0;
```

Comment:
nit: it looks like this constant is only used as a placeholder for the `to` field in the `snapshot` method. could we update either the variable name or docstring to specify this? the current description is a bit confusing and might lead to this constant being used in the wrong place

Reply:
sure good call, I'll update the docstring because I think `UNUSED` reads better inline when this variable is used